tools/data_converter/s3dis_data_utils.py

```diff
@@ -24,13 +24,21 @@ def __init__(self, root_path, split='Area_1'):
             'ceiling', 'floor', 'wall', 'beam', 'column', 'window', 'door',
             'table', 'chair', 'sofa', 'bookcase', 'board', 'clutter'
         ]
-        self.cat2label = {cat: self.classes.index(cat) for cat in self.classes}
-        self.label2cat = {self.cat2label[t]: t for t in self.cat2label}
+
+        # Use all 13 classes for segmentation.
         self.cat_ids = np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12])
         self.cat_ids2class = {
             cat_id: i
             for i, cat_id in enumerate(list(self.cat_ids))
         }
+        # Following https://arxiv.org/abs/2006.12356, use 5 furniture classes
+        # for detection: table, chair, sofa, bookcase, board.
+        self.furniture_cat_ids = np.array([7, 8, 9, 10, 11])
+        self.furniture_cat_ids2class = {
```

Comment:
We may rename `cat_ids` to `seg_cat_ids` and `furniture_cat_ids` to `det_cat_ids` so that their names are more easy to understand. Do you think that's acceptable?